ci/build.yml

```diff
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.16.3
+    tag: 1.16.7
```

Comment:
As this is a release, we don't need to do anything here now - but general fyi we should be moving to go version 1.17.1 now :)

Reply:
fully aware of that, but the work in `develop` was done before that came out, AIUI